seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -4182,14 +4182,14 @@
     },
     {
       "id": "56533eb9ac21ba0edf2244c7",
-      "title": "Accessing Objects Properties with the Dot Operator",
+      "title": "Accessing Objects' Properties with the Dot Operator",
```

Comment:
Eric, I agree with you on using Object as an adjective here (and removing the 's') for the titles. Then we may want to make the challenge description part consistent, or rephrase things to avoid using `object` as a noun. Having it used as both an adjective and a noun may confuse folks who aren't fluent in English. For example, re-word the sentence:

"Here is a sample of using the dot operator (`<code>.</code>`) to read the properties of `<code>myObj</code>`:"

In the instructions, I think we could either keep the sentence how it was (`object` is an adjective and consistent with the title), or drop the two `object's` completely (it's clear by this point in the challenge we're talking about object properties, vs. some other type of property). I don't feel strongly either way.